go/cmd/dolt/commands/cherry-pick.go

```diff
@@ -154,7 +154,13 @@ hint: commit your changes (dolt commit -am \"<message>\") or reset them (dolt re
 		return fmt.Errorf("error: failed to set @@dolt_force_transaction_commit: %w", err)
 	}
 
-	q, err := dbr.InterpolateForDialect("call dolt_cherry_pick(?)", []interface{}{cherryStr}, dialect.MySQL)
+	interfaceArgs := make([]interface{}, 0, len(args))
+	for _, arg := range args {
+		interfaceArgs = append(interfaceArgs, arg)
+	}
+
+	placeholders := strings.Join(make([]string, len(args)), "?, ") + "?"
```

Comment:
Not wild about this. maybe a little helper function?

```
func generateQuestionMarks(count int) string {
    if count <= 0 {
        return ""
    }
    return strings.Repeat("?,", count-1) + "?"
}
```

`placeholders := generateQuestionMarks(len(args))`

We should probably have a helper function to convert arrays of strings into []interface{} instances. Kind of surprised I'm not finding that.

Reply:
Good call, updated in 7f4810594a80d403e14dfc09cc3ac8c473f41b0c